assembl/static2/js/app/components/form/fieldArrayWithActions.jsx

```diff
@@ -151,17 +151,17 @@ export class Fields extends React.PureComponent<FieldsProps> {
       onUp,
       onDown
     } = this.props;
-    /* Hack to fix issue with richtext field:
-    when clicking on delete action, the value of richtext field becomes '' and so
-    new EditorState.createEmpty() is created triggering onChange event on first render of the field)
-    preventing the whole block to be removed */
-    // console.log(fields.length);
-    const removedIndex = this.removedIndex;
-    if (this.removedIndex) {
-      setTimeout(() => {
-        this.removedIndex = null;
-      }, 2000);
-    }
+    // /* Hack to fix issue with richtext field:
+    // when clicking on delete action, the value of richtext field becomes '' and so
+    // new EditorState.createEmpty() is created triggering onChange event on first render of the field)
+    // preventing the whole block to be removed */
+    // // console.log(fields.length);
+    // const removedIndex = this.removedIndex;
+    // if (this.removedIndex) {
+    //   setTimeout(() => {
+    //     this.removedIndex = null;
+    //   }, 2000);
+    // }
```

Comment:
The hack created a bug with the delete action of fieldArrayWithActions. Sometimes It deleted 2 phases instead of one. Commenting the hack resolved the bug. We need to find another hack !